aws-arn.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:   2.2
 name:            aws-arn
-version:         0.3.2.0
+version:         0.3.3.0
```

Comment:
Normally I'd say do this just before release but that should happen shortly after merge so it's fine. I think change the `Unreleased` line to include the new version and date.